_posts/0000-01-02-olloffgit.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
This is awesome!
![cute-robot](https://user-images.githubusercontent.com/104035874/166102986-6e7c8f29-dbc2-46c9-9895-895e1d016e07.jpg)